src/app/shared/issue-tables/issue-tables-paginated-data.ts

```diff
@@ -0,0 +1,14 @@
+import { MatPaginator } from '@angular/material';
+import { Issue } from '../../core/models/issue.model';
+
+export function setAndGetPaginatedData(paginator: MatPaginator, data: Issue[]): Issue[] {
```

Comment:
I think a shorter name: `paginateData` would suffice, instead of `setAndGetPaginatedData`.

The current file name is a slight misnomer (as the module does not contain data), I think `issue-paginator.ts` will be clearer.

Reply:
Thanks for the suggestion!